gcloud-java-compute/src/main/java/com/google/gcloud/compute/DiskType.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+import com.google.api.client.util.DateTime;
```

Comment:
Nit, I think we should avoid depending `com.google.api.client.util` as much as possible.
The only one exception is `Data` (which is part of the API).
In this case we should use Joda (until Java8).
I see that we already use it in other [places](https://github.com/GoogleCloudPlatform/gcloud-java/search?utf8=%E2%9C%93&q=%22com.google.api.client.util%22&type=Code), so this should be done in a separate cleanup PR (even more so for Lists which should be replaced with guava).